packages/mask/src/plugins/ITO/SNSAdaptor/ClaimAllDialog.tsx

```diff
@@ -375,6 +381,7 @@ export function ClaimAllDialog(props: ClaimAllDialogProps) {
                                 ChainId.Arbitrum,
                                 ChainId.xDai,
                                 ChainId.Fantom,
+                                ChainId.Avalanche,
```

Comment:
https://github.com/DimensionDev/Maskbook/pull/5348/files#r776580361

use extracted constant replace the `chains` hardcoded array.

Reply:
okay

Reply:
done